README.md

```diff
@@ -146,6 +146,12 @@ grpcurl -d @ grpc.server.com:443 my.custom.server.Service/Method <<EOM
 EOM
 ```
 
+Sometimes it might be necessary to read the payload from a file. This worked for some users:
+
+```shell
+grpcurl -d "$(cat my.json)" grpc.server.com:443 my.custom.server.Service/Method
```

Comment:
I think the more straight-forward way, that doesn't require this particular feature in your shell, is to use the `@` value as the input, very much like the example above.
```suggestion
grpcurl -d @ grpc.server.com:443 my.custom.server.Service/Method < my.json
```